urlBuilder.test.ts

```diff
@@ -0,0 +1,47 @@
+import { multiUrlGenerator } from "./multiUrlGenerator";
+//"This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript."
```

Comment:
It should be possible to run tests in TS files without the need to transpile it:
https://basarat.gitbook.io/typescript/intro-1/jest